File: AOJ/ALSD1/01_C/a.cpp
#include<iostream>
using namespace std;

bool isPrime(int v){
    if(v == 1) return false;
    if(v == 2) return true;
    for(int i=2; i*i<=v; i++){
        if(v % i == 0) return false;
    }
    return true;
}

int main(){
    int n; cin >> n;

    long long ans = 0;
    while(n--){
        int v; cin >> v;
        if(isPrime(v)) ans++;
    }
    cout << ans << endl;
}

File: AOJ/ALSD1/01_C/a.py
import math
def isPrime(n: int):
    if n == 1: return False
    if n == 2: return True
    for i in range(2, int(math.sqrt(n))+1):
        if n % i == 0: return False
    return True

n = int(input())
ans = 0
for _ in range(n):
    num = int(input())
    if(isPrime(num)): ans += 1
print(ans)
